### PR TITLE
refactor test for ProductFilterContainer component

### PR DESCRIPTION
**Pull request by Yevhendzyuba**
## Description

Refactor test for ProductFilterContainer component by RTL.

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/products-filters-container/tests/product-filter-container.spec.js [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
       <ProductsFiltersContainer
-        productFilter=''
-        list={[]}
-        categories=''
-        filterHandler=''
-        clearFilter=''
-        filterName=''
-        filterAction=''
-        labels={[]}
+        productFilter={['6043bdeb3e06ad3edcdb7b2d']}
```
Comment (afanasieva-anastasiia) [inline src/containers/products-filters-container/tests/product-filter-container.spec.js:13]: Сan you take this out to constants?

**Review by afanasieva-anastasiia — CHANGES_REQUESTED** [review]

File: src/containers/products-filters-container/tests/product-filter-container.spec.js (hunk 1 above)
Comment (Yevhendzyuba) [reply src/containers/products-filters-container/tests/product-filter-container.spec.js]: Thanks, fixed

File: src/tests/unit/containers/products-filters-container/products-filters-container.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        productFilter={categoryId}
+        list={categoriesList}
+        categories={categoriesInfo}
+        filterHandler={handler}
+        filterName={blockHeading}
+      />
+    );
+  });
+
+  test('Checkbox click', () => {
```
Comment (SirAndrii) [inline src/tests/unit/containers/products-filters-container/products-filters-container.spec.js:44]: describe what you expect in that test 'it should .....'

File: src/tests/unit/containers/products-filters-container/products-filters-container.spec.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    const checkbox = screen.getAllByRole('checkbox')[0];
+    fireEvent.click(checkbox);
+    expect(checkbox).toBeChecked();
```
Comment (SirAndrii) [inline src/tests/unit/containers/products-filters-container/products-filters-container.spec.js:47]: add empty line before expect for readability

File: src/tests/unit/containers/products-filters-container/products-filters-container.spec.js (hunk 2 above)
Comment (Yevhendzyuba) [reply src/tests/unit/containers/products-filters-container/products-filters-container.spec.js]: Fixed

File: src/tests/unit/containers/products-filters-container/products-filters-container.spec.js (hunk 3 above)
Comment (Yevhendzyuba) [reply src/tests/unit/containers/products-filters-container/products-filters-container.spec.js:48]: Fixed

**Review by afanasieva-anastasiia — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**